uiServices/src/homepage.go

```diff
@@ -21,6 +25,16 @@ func StartUiMode() {
 
 func GetRouter() *chi.Mux {
 	r := chi.NewRouter()
+	cors := cors.New(cors.Options{
+		AllowedOrigins:   []string{"*"},
```

Comment:
were there issues with the project that required this change?

Reply:
Yeah this makes doing the front end dev way better and quicker.